infra/core/monitor/monitoring.bicep

```diff
@@ -7,33 +7,43 @@ param tags object = {}
 @allowed([ 'Enabled', 'Disabled' ])
 param publicNetworkAccess string = 'Enabled'
 
-module logAnalytics 'loganalytics.bicep' = {
+module logAnalytics 'br/public:avm/res/operational-insights/workspace:0.4.0' = {
   name: 'loganalytics'
   params: {
     name: logAnalyticsName
     location: location
     tags: tags
+    skuName: 'PerGB2018'
+    dataRetention: 30
     publicNetworkAccessForIngestion: publicNetworkAccess
```

Comment:
I'm surprised that the avm has a parameter named publicNetworkAccessForIngestion ? How is that working?

Reply:
It's a parameter of the log analytics workspace and the application insights account anyway https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/operational-insights/workspace/main.bicep#L192-L193

Reply:
Ohh, I thought it was one we made up ("ingestion" threw me off). Got it, thanks